install.sh

```diff
@@ -2,5 +2,7 @@
 set -e
 
 pushd ../
-ln -sf sha3/marshal-configs/*.json .
```

Comment:
There is no need to change this glob.  This results in a literal `*.json` only when there are no matches and the `nullglob` option is set in Bash (the default).

If you want to make this more robust, I suggest checking for `[ "${PWD##*/}" = 'sha3' ]`.